app/javascript/packages/document-capture/components/location-collection-item.tsx

```diff
@@ -2,33 +2,47 @@ import { Button } from '@18f/identity-components';
 import { useI18n } from '@18f/identity-react-i18n';
 
 interface LocationCollectionItemProps {
+  distance?: string;
   formattedCityStateZip: string;
   handleSelect: (event: React.FormEvent<HTMLInputElement>, selection: number) => void;
-  name: string;
+  name?: string;
+  phone?: string;
   saturdayHours: string;
   selectId: number;
   streetAddress: string;
   sundayHours: string;
+  tty?: string;
   weekdayHours: string;
 }
 
 function LocationCollectionItem({
+  distance,
   formattedCityStateZip,
   handleSelect,
   name,
+  phone,
   saturdayHours,
   selectId,
   streetAddress,
   sundayHours,
+  tty,
   weekdayHours,
 }: LocationCollectionItemProps) {
   const { t } = useI18n();
+  const numericDistance = distance?.split(' ');
 
   return (
     <li className="location-collection-item">
       <div className="usa-collection__body">
         <div className="display-flex flex-justify">
-          <h3 className="usa-collection__heading">{name}</h3>
+          {distance && numericDistance && (
```

Comment:
If distance is falsey then numericDistance will be too

```suggestion
          {numericDistance && (
```